proposals/Shadow-DOM-Cascade-Order-in-v1.md

```diff
@@ -116,11 +116,11 @@ Note that `host4` is in **host1's shadow tree**.
 
      - `... { color: #000; }`
 
-2.   A selectors declared in the same node tree (*host1' shadow tree*):
+2.   A selectors declared in the same node tree (*host1's shadow tree*):
```

Comment:
lol. We can fix that easily. :)
I wrote this document in an hour, in Sapporo Convention Center. You might find more and more minor mistakes.